### DISCUSSION TITLE
No confirm message when delete a connection

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#5152] [Feature Request]
@rustdesk 

### Bug Description

When you select the deletion of a connction appeara a small window with title as "delete" and "OK" /"Cancel" without a maessage liek "Are you sure to delete the connection"?

I beleiev that could be useful ammessage in the window like "Are you sure to delete the connection".

### How to Reproduce

Delete a connection

### Expected Behavior

Add a string to confirtm the deletion

### Operating system(s) on local side and remote side

Windows 10 22 HH2 64bit italian

### RustDesk Version(s) on local side and remote side

night build 26.07.2023

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/1262554/c5f2c5a8-d88d-48d2-83de-2915f8c2e9a5)

**Comment by rustdesk** [comment]
A simple PR is welcome.

**Comment by bovirus** [comment]
@rustdesk 
Thanks.
I didn't know how to add the message for conformation.
